kolibri/core/analytics/pskolibri/__init__.py

```diff
@@ -161,7 +155,6 @@ def cpu_percent():
     utilization as a percentage.
     """
     global _last_cpu_times
-    global _last_per_cpu_times
```

Comment:
yeah, this is not needed anymore as we are not calculating cpu usage per core, but per system.